Touch Bar Simulator/Constants.swift

```diff
@@ -12,4 +12,5 @@ extension Defaults.Keys {
 	static let lastFloatingPosition = OptionalKey<CGPoint>("lastFloatingPosition")
 	static let dockBehavior = Key<Bool>("dockBehavior", default: false)
 	static let lastWindowDockingWithDockBehavior = Key<TouchBarWindow.Docking>("windowDockingWithDockBehavior", default: .dockedToTop)
+	static let launchAtLogin = Key<Bool>("launchAtLogin", default: false)
```

Comment:
We don't actually need to save it to UserDefaults. `LaunchAtLogin` keeps track of whether it's enabled or not itself.